code/game/objects/items/robot/robot_items/robot_rack.dm

```diff
@@ -68,7 +68,11 @@
 	. = ..()
 
 /obj/item/robot_rack/bed/update_icon()
-	icon_state = "borgbed_[length(held) > 0 ? "stored" : "deployed"]"
+	icon_state = "[initial(icon_state)][length(held) > 0 ? "stored" : "deployed"]"
```

Comment:
wouldn't this make it invisible before `update_icon` is called? or is `update_icon` already called somewhere upon creation?

Reply:
It is called when the rack spawns the initial icon

Reply:
👌